frontend/src/components/requisitionForm/RequisitionForm.tsx

```diff
@@ -0,0 +1,215 @@
+    value: project.name
+  }));
+
+  const vendorOptions = loading ? [] : data.vendors.map((vendor: Vendor) => ({
+    label: vendor.name,
+    value: vendor.id
+  }));
+
+  const onFinish = (values: any) => {
+    console.log("Form Success:", values);
+  };
+
+  const onFinishFailed = (errorInfo: any) => {
+    console.log("Failed:", errorInfo);
+  };
+
+  const halfLayout = {
+    xs: 24, sm: 12, md: 8, lg: 6, xl: 6
+  };
+  const fullLayout = {
+    xs: 24, sm: 24, md: 16, lg: 12, xl: 12
+  };
+
+  return (
+    <>
+      <Title level={2}>Create Requisition</Title>
+      <Form
+        name="create"
+        initialValues={{ items: [{}] }}
+        onFinish={onFinish}
+        onFinishFailed={onFinishFailed}
+        layout="vertical"
+        autoComplete="off"
+        scrollToFirstError
+      >
+        <Row gutter={[32, 8]} justify="center">
+          <Col {...halfLayout}>
+            <Form.Item
+              name="headline"
+              rules={[RULES.requiredRule]}
+              label="Headline"
+            >
+              <Input placeholder="Giant Outdoor Games" />
+            </Form.Item>
+          </Col>
+
+          <Col {...halfLayout}>
+            <Form.Item
+              name="project"
+              rules={[RULES.requiredRule]}
+              label="Project"
+            >
+              <Select options={projectOptions} showSearch optionFilterProp="label" loading={loading} />
+            </Form.Item>
+          </Col>
+        </Row>
+
+        <Row gutter={[32, 8]} justify="center">
+          <Col {...fullLayout}>
+            <Form.Item
+              name="description"
+              rules={[RULES.requiredRule]}
+              label={(
+                <span>
+                  {"Description "}
+                  <Tooltip title="The description of what you want to order and why you need it">
+                    <QuestionCircleOutlined />
+                  </Tooltip>
+                </span>
+              )}
+            >
+              <TextArea
+                autoSize={{ minRows: 2 }}
+                placeholder="This will help spice up our venue, and add the pizzazz that will draw students from around the world!"
+              />
+            </Form.Item>
+          </Col>
+        </Row>
+
+        <Row gutter={[32, 8]} justify="center">
+          <Col {...halfLayout}>
+            <Form.Item
+              name="vendor"
+              rules={[RULES.requiredRule]}
+              label="Vendor"
+            >
+              <Select options={vendorOptions} showSearch optionFilterProp="label" loading={loading} />
+            </Form.Item>
+          </Col>
+
+          <Col {...halfLayout}>
+            <Form.Item
+              name="paymentRequiredBy"
+              rules={[RULES.requiredRule]}
+              label={(
+                <span>
+                  {"Payment Required By "}
+                  <Tooltip title="The date you need the payment by (Should be at least 3 days from today)">
```

Comment:
should probably put a TODO here to fill in the actual policy, but great that you're even thinking of this kind of stuff 💯

Reply:
Yeah, you're right Ill add it in